packages/core/test/acceptance/application.acceptance.ts

```diff
@@ -49,6 +49,10 @@ describe('Bootstrapping the application', () => {
       expect(app.find('controllers.*').map(b => b.key)).to.eql([
         'controllers.ProductController',
       ]);
+
+      expect(app.findByTag('controller').map(b => b.key)).to.eql([
```

Comment:
+1

Please consider using a shared helper `findKeysByTag` described in one of my comments below, perhaps we should make this function a member method of the Context class?